modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
++
+[IMPORTANT]
+====
+Breaking change: The Status label has been removed from the `eventlistener_http_duration_seconds_*` metric.
+Remove queryies that are based on the status label.
+====
+
+=== Tekton Triggers v0.15.0
+
+* With the current release, the `EventListener` sends a response before Triggers finishes processing. (link:https://github.com/tektoncd/triggers/pull/1132[#1132])
++
+[IMPORTANT]
+====
+Breaking change: With this change, the `EventListener` stops responding with a `201 Created` status code when it creates resources. Instead, it responds with a `202 Accepted` response code.
+====
+
+* The current release removes the `podTemplate` field from the `EventListener`. (link:https://github.com/tektoncd/triggers/pull/1118[#1118])
++
+[IMPORTANT]
+====
+Breaking change: The `podTemplate` field, which was deprecated as part of link:https://github.com/tektoncd/triggers/pull/1100[#1100], has been removed.
+====
+
+* The current release removes the deprecated `replicas` field from the `EventListener` specification. (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
+[id="removing-home-workingdir-values-1-6_{context}"]
+=== Removing the HOME="/tekton/home" and workingDir="/workspace" values from Steps
+// TBD - Where does this release note belong?
+
+In {pipelines-title} 1.6, the values of `HOME="/tekton/home"` and `workingDir="/workspace"` are removed from the `Steps` specification.
+
+Instead, {pipelines-title} sets `HOME` and `workingDir` to the values defined by your Steps containers. You can override these values in your `Steps` YAML file.
+
+To get the older behavior, you can change the following fields in `TektonConfig` to `false`. Their default values are `true`.
+
+[source,yaml]
+----
+spec:
+  pipeline:
+    disable-working-directory-overwrite: false
+    disable-home-env-overwrite: false
+----
+
+[IMPORTANT]
+====
+The feature flags are temporarily available. These flags are candidates for complete removal from a future minor release of the upstream product at least nine months from when this release goes live.
+====
+
+(link:https://issues.redhat.com/browse/SRVKP-1465[SRVKP-1465])
+
+[id="known-issues-1-6_{context}"]
+== Known issues
+
+* When you run Maven and Jib-Maven cluster tasks on IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) clusters, set the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
+// issue # is unknown.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `s2i-dotnet` cluster task is unsupported.
+// issue # is unknown.
+
+* Before you install catalog tasks on IBM Power Systems, IBM Z, and LinuxONE using `tkn hub`, make sure to check task information about these platforms. For example, using the following command: `tkn hub info task <name>`
```

Comment:
what info are we looking for here? What does it indicate?

Reply:
@SNiemann15 Can we explain this a bit more?

Reply:
@rolfedh I got feedback from Yulia our SME that this is needed because some tasks don't run on P/Z. Yulia also sent me an example output but it's pretty large not sure if you want to have that in the Release Notes. 
Is it ok like this or do you think we need more information? 

* Before you install `tekton` catalog tasks on IBM Power Systems (ppc64le), IBM Z, and LinuxONE(s390x) using `tkn hub`, verify if the task can be executed on these platforms.
To check if `ppc64le` and `s390x` are listed in the "Platforms" section of the task information, you can run the following command: `tkn hub info task <name>`

Reply:
Updated. Thanks.